gui/package.json

```diff
@@ -9,39 +9,39 @@
     "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'"
   },
   "dependencies": {
-    "@types/c3": "0.7.4",
-    "@types/moment-timezone": "0.5.12",
-    "@types/vuelidate": "0.7.8",
+    "@types/c3": "^0.7.5",
```

Comment:
^ should not be added for any of the versions. Please remove for all

Reply:
remove all cap sign from both file